docs/components/select.md

```diff
@@ -540,6 +540,7 @@ You can pass any input attribute and it will get properly bound to the element.
 ## Slots
 
 - `item-template` - The template for each item in the dropdown list
+- `selected-item` - Slot for customizing selected item appearance
```

Comment:
suggestion: can we name this more similarly to the unselected slot `item-template`?

```suggestion
- `selected-item-template` - Slot for customizing selected item appearance
```

Reply:
Good idea!